daft/execution/physical_plan.py

```diff
@@ -777,7 +777,7 @@ def sort(
                 partial_metadatas=None,
             )
             .add_instruction(
-                instruction=execution_step.Sample(sort_by=sort_by),
+                instruction=execution_step.Sample(size=20, sort_by=sort_by),
```

Comment:
We now have some [centralized configs](https://github.com/Eventual-Inc/Daft/blob/main/daft/context.py#L204), might be a good place to put this constant.

Reply:
added the constant: `sample_size_for_sort` in DaftExecutionConfig here https://github.com/Eventual-Inc/Daft/pull/1770/files#diff-9feb783dcfde34fbbab50546b081d87cfb98ec050b89cfbe6ff9fe64e1404029R37 . I didn't add this as a parameter in `set_execution_config' though, I assume that this isn't something that the user would need to change. but lmk if I'm wrong!